lib/wickedbase.pm

```diff
@@ -554,9 +580,8 @@ sub wait_for_dhcpd {
 
 sub ifbind {
     my ($self, $action, $interface) = @_;
-    my $cmd = "./ifbind.sh $action $interface";
-    record_info('bind', $cmd);
-    assert_script_run($cmd);
+    my $cmd = 'bash ' . WICKED_DATA_DIR . "/wicked/ifbind.sh $action $interface";
```

Comment:
not sure about what the difference but I always was using 
```suggestion
    my $cmd = 'source ' . WICKED_DATA_DIR . "/wicked/ifbind.sh $action $interface";
```

when I need to do something like this . But not insist just saying

Reply:
I use bash command, but I guess if you use source, it depends on the shebang, which in this case `#!/bin/bash` it would be the same thing.

Reply:
ok